contributing/development/handling_compatibility_breakages.rst

```diff
@@ -100,7 +100,7 @@ like ``core/math/a_star_grid_2d.compat.inc`` in this case:
 
     #endif // DISABLE_DEPRECATED
 
-Unless the change in compatibility is complex, the compatibility method should simply call the modified method directly,
+Unless the change in compatibility is complex, the compatibility method should call the modified method directly,
```

Comment:
I'd argue this one is not in the spirit of the rule, as the meaning here does not seem to be "this is obvious and/or simple to you, reader", but rather "the compat method should do a simple (as in direct) call in this case, instead of something more involved".

Reply:
The rule also covers empty adverbs like this!

I'd argue that if the reader should *directly* call the method, you could use the world "directly" instead of "simply" here.